ui/src/components/VoxettaSkipButton.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LitElement, html, css} from 'lit-element';
+
+import * as promptApi from '../utils/PromptApiService';
+
+import {Icon} from '@material/mwc-icon';
+
+export class VoxettaSkipButton extends LitElement {
+    static get properties() {
+        return {
+            prompt: {type: String},
+            state: {type: String},
+        };
+    }
+
+    static get styles() {
+        return css`
+            button {
+                background-color: white;
+                border: none;
+                color: #a0a0a0;
+                cursor: pointer;
+                font-family: 'Roboto';
+                width: 50px; 
+            }
+            button:hover {
+                background-color: #dcdcdc;
+            }
+        `;
+    }
+
+    render() {
+        return html`
+            <button @click="${this.handleSkip}">
+                <mwc-icon>skip_next</mwc-icon>
+                SKIP
```

Comment:
nit, you can apply the upper case with css

```css
button {
  text-transform: capitalize;
}
```

Reply:
adjusted